google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/datacatalog_tag_template_factory.py

```diff
@@ -125,29 +125,28 @@ def make_tag_template_for_folder(self) -> TagTemplate:
                                        is_required=True,
                                        order=11)
 
-        self._add_primitive_type_field(tag_template=tag_template,
-                                       field_id='name',
-                                       field_type=self.__STRING_TYPE,
-                                       display_name='Name',
-                                       is_required=True,
-                                       order=10)
-
         self._add_primitive_type_field(tag_template=tag_template,
```

Comment:
More related to previous PR's, but taking a better look at the tag fields, some inconsistencies that should be verified:

```
self._add_primitive_type_field(tag_template=tag_template,
                                       field_id='owner_username',                                       
                                       field_type=self.__STRING_TYPE,                                       
                                       display_name='Owner username',                                       
                                       is_required=True,                                       
                                       order=9)
```
`owner_username` is_required for `make_tag_template_for_dashboard` but not for `make_tag_template_for_folder`.

And in the tag field code:
```
owner = dashboard_metadata.get('ownerData')        
if owner:
```

The owner can be null, so the required field would throw an error, is this the intended behavior?

Reply:
You're right, @mesmacosta.Thanks!
Dashboards always have owners, AFAIK, but their data may not be retrieved if the Sisense account used to run the connector does not have the appropriate access. This is the case in which the owner would be null. I've removed the `is_required` flag from the `Owner username` field.